rocAL/rocAL/source/rali_api_meta_data.cpp

```diff
@@ -77,13 +77,21 @@ RALI_API_CALL raliCreateVideoLabelReader(RaliContext p_context, const char* sour
 }
 
 RaliMetaData
-RALI_API_CALL raliCreateCOCOReader(RaliContext p_context, const char* source_path, bool is_output){
+RALI_API_CALL raliCreateCOCOReader(RaliContext p_context, const char* source_path, bool is_output, bool keypoint, float sigma , int pose_output_width , int pose_output_height){
```

Comment:
This function has been changed in header file. This is the normal coco reader which does not have any key points variables. 
Make changes to this function

Reply:
Made modification as per the comment